text/0000-sawtooth-governance.md

```diff
@@ -0,0 +1,331 @@
+- Feature Name: governance
```

Comment:
the file is called sawtooth-governance should be grid-governance.

Reply:
Fixed